infrastructure/k8s/services/queue.yaml

```diff
@@ -0,0 +1,113 @@
+project_name: "taskcluster-queue"
+secrets:
+  artifact_region: ".Values.artifactRegion"
+  aws_access_key_id: ".Values.awsAccessKeyId"
+  aws_secret_access_key: ".Values.awsSecretAccessKey"
+  azure_account_id: ".Values.tfAzureAccount"
+  azure_account_key: ".Values.azureAccountKey"
+  blob_artifact_region: ".Values.blobArtifactRegion"
+  force_ssl: "false"
+  monitoring_enable: "true"
+  node_env: "production"
+  private_artifact_bucket: ".Values.privateArtifactBucket"
+  private_blob_artifact_bucket: ".Values.privateBlobArtifactBucket"
+  public_artifact_bucket: ".Values.publicArtifactBucket"
+  public_blob_artifact_bucket: ".Values.publicBlobArtifactBucket"
```

Comment:
For all 4 of these [buckets](https://github.com/sciurus/taskcluster/blob/cloudops/infrastructure/terraform/main.tf#L17-L102), have terraform [output](https://github.com/sciurus/taskcluster/blob/cloudops/infrastructure/terraform/outputs.tf) the bucket IDs and then read them back in as `tf` prefixed variables.

Reply:
https://github.com/mozilla-services/cloudops-infra/pull/1105 fixes